src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/client/PrisonApiClient.kt

```diff
@@ -110,12 +110,12 @@ class PrisonApiClient(
     }
   }
 
-  fun retrieveAgency(agencyId: String): Mono<Agency> {
+  fun retrieveAgency(agencyId: String, activeOnly: Boolean = false): Mono<Agency> {
```

Comment:
Do we need to call it with `activeOnly = true` from anywhere? If not, maybe it's better to just keep it set to false within this function and not bother with extra parameters?